File: Proyecto.MVC/Models/Incidencia.cs
using System.Collections.Generic;

namespace Proyecto.MVC.Models
{
    public class Incidencia
    {
        public class IndexLoad
        {
            public List<Incidencia> Incidencias { get; set; }
            public List<EstadoIncidencia> EstadoIncidencias { get; set; }
            public List<TipoIncidencia> TipoIncidencias { get; set; }
        }

        public int Id { get; set; }
        public long Identificacion { get; set; }
        public string PrimerNombre { get; set; }
        public string SegundoNombre { get; set; }
        public string PrimerApellido { get; set; }
        public string SegundoApellido { get; set; }
        public string Direccion { get; set; }
        public string Telefono { get; set; }
        public string Correo { get; set; }
        public EstadoIncidencia EstadoIncidencia { get; set; }
        public TipoIncidencia TipoIncidencia { get; set; }
    }
}

File: Proyecto.MVC/Scripts/Views/Incidencia.js
$(document).ready(function () {
    $("#btnGuardarIncidencia").click(function () {
        GuardarIncidencia();
    });
});

function GuardarIncidencia() {

    if ($("#txtIdentificacion").val() == "") {
        alert("Ingrese identificación");
        return;
    }

    $.ajax({
        data: {
            "Identificacion": $("#txtIdentificacion").val(),
            "PrimerNombre": $("#txtPrimerNombre").val(),
            "SegundoNombre": $("#txtSegundoNombre").val(),
            "PrimerApellido": $("#txtPrimerApellido").val(),
            "SegundoApellido": $("#txtSegundoApellido").val(),
            "Direccion": $("#txtDireccion").val(),
            "Telefono": $("#txtTelefono").val(),
            "Correo": $("#txtCorreo").val(),
            "EstadoIncidencia.Id": $("#ddlEstadoIncidencia").val(),
            "TipoIncidencia.Id": $("#ddlTipoIncidencia").val(),
        },
        type: "POST",
        dataType: "json",
        url: "/Incidencia/Create",
        success: function (data) {
            alertify.success(data.Mensaje);
        },
        error: function (jqXHR, error, errorThrown) {
            var responseTitle = $(jqXHR.responseText).filter('title').get(0);
            alertify.error($(responseTitle).text());
        }
    });

    $("#NuevoIncidencia").modal('hide');
};